vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/Dashboard.java

```diff
@@ -244,6 +244,7 @@ private JsonArray createItemsJsonArray() {
         for (DashboardWidget widget : widgets) {
             JsonObject jsonItem = Json.createObject();
             jsonItem.put("nodeid", getWidgetNodeId(widget));
+            jsonItem.put("colspan", widget.getColspan());
```

Comment:
This isn't tested. Should there be a `getColspan()` in `DashboardWidgetElement`?

Reply:
There are unit tests for this in `DashboardTest`, which checks the `colspan` through the `items` set to the client. That's why I didn't add any ITs. Do you think we should still add ITs?

`getColspan()` in `DashboardWidgetElement` makes sense. I will add that.

Reply:
Added `getColspan()` to `DashboardWidgetElement`. It is quite ugly, but it works. I couldn't see a better way to implement this. Any suggestions are welcome.

Reply:
No need to access the wrapper if you use `getComputedStyle`:

```java
public Integer getColspan() {
    var colspanStr = getComputedCssValue("--vaadin-dashboard-item-colspan");
    return colspanStr.isEmpty() ? null : Integer.valueOf(colspanStr);
}

private String getComputedCssValue(String propertyName) {
    return (String) executeScript(
            "return getComputedStyle(arguments[0]).getPropertyValue(arguments[1]);",
            this, propertyName);
}
```

Reply:
> There are unit tests for this in DashboardTest, which check the colspan through the items sent to the client.

Ah, somehow I didn't catch these tests failing with the line removed. All good regarding testing the component logic then.
However, nothing is currently testing the `getColspan()` API of `DashboardWidgetElement`.

Reply:
Thanks, I applied your computed style suggestion to get the colspan. For `getColspan()` being untested, I was not sure if we should add anything to test it. But I added 2 simple ITs that should cover it.